include/zephyr/ipc/icmsg.h

```diff
@@ -145,3 +145,15 @@ int icmsg_send(const struct icmsg_config_t *conf,
  *  @retval 0 on success.
  */
 int icmsg_clear_tx_memory(const struct icmsg_config_t *conf);
+
+/** @brief Clear memory in RX buffer.
+ *
+ *  This function is intended to be called at an early stage of boot process,
+ *  before the instance is initialized and before the remote core has started.
+ *
+ *  @param[in] conf Structure containing configuration parameters for the icmsg
+ *                  instance being created.
+ *
+ *  @retval 0 on success.
+ */
+int icmsg_clear_rx_memory(const struct icmsg_config_t *conf);
```

Comment:
can we just have one single `icmsg_prepare_memory` or something like that instead of having one for rx and one for tx?

Reply:
I'm going to remove them both in one of the following PRs. This functionality seem redundant with the new handshake recently introduced by @rkel